index.html

```diff
@@ -1,54 +1,187 @@
-           </div>
-           <div class="hamburger">
-             <a href="#"><img class="icon-bar" src="./images/hamburger.png" alt="Icon"></a>
-           </div>
-           <div class="desktop-nav">
-            <ul>
-                <li>Home</li>
-                <li>About</li>
-                <li>Blog</li>
-            </ul>
-           </div>
-       </nav>
-       <div class="header-text">
-           <h3>Lorem ipsum dolor sit amet.</h3>
-           <h1>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dicta!</h1>
-           <div class="border-text">
-               <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. 
-                   Cupiditate magnam delectus vel minima. Dignissimos, magnam!
-                </p>
-           </div>
-           <div class="header-date">
-            <h1>2022.04.11(MON) ~ 13(WED)</h1>
-            <p>@iHub Kenya, Caren Shopping Centre.</p>
-           </div>
-       </div>
-   </header>
-   <section class="program">
-       <div class="program-header">
-            <h2>Main Program</h2>
-            <hr>
-       </div>
-       <div class="program-list">
-           <div class="program-list-items">
-               
-           </div>
-       </div>
-   </section>
+  <div class="header-container">
+    <header>
+      <nav class="social">
+        <ul>
+          <li>
+            <a href="https://twitter.com/odhiambo_ed"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/edward-odhiambo-6a462a21b/"><i class="fa-brands fa-linkedin"
+               ></i></a>
+          </li>
+          <li>
+            <a href="https://github.com/white3d"><i class="fa-brands fa-github"></i></a>
+          </li>
+          <li><a href="">English</a></li>
+          <li><a href="">French</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <div class="logo">
+          <a href="index.html"><img src="./images/logo.png" alt="IDREALITY" /></a>
+        </div>
+        <div class="hamburger">
+          <a href="#"><img class="icon-bar" src="./images/hamburger.png" alt="Icon" /></a>
+        </div>
+        <div class="desktop-nav">
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a class="nav-btn" href="#">Share Your Mind</a></li>
+          </ul>
+        </div>
+      </nav>
+      <nav id="mobile-nav" class="mobile-nav">
+        <span class="mobile-close"><i class="fas fa-times"></i></span>
+        <ul>
+          <li><a href="index.html" onclick="menuLink()">Home</a></li>
+          <li><a href="about.html" onclick="menuLink()">About</a></li>
+          <li><a href="#" onclick="menuLink()">Blog</a></li>
+          <li><a class="nav-btn" href="#" onclick="menuLink()">Share Your Mind</a></li>
+        </ul>
```

Comment:
- [ ] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157931712-d593152e-a8e1-49f4-9a1f-36c382b6fe54.png)